index.bs

```diff
@@ -187,7 +186,101 @@ If the console is not open when the printer operation is called,
 implementations should buffer messages to show them in the future up to an
 implementation-chosen limit (typically on the order of at least 100).
 
-<h4 id="object-formats">Common Object Formats</h4>
+<h4 id="loglevel-severity">Indicating <var>logLevel</var> severity</h4>
+
+Each {{console}} method uses a unique value for the logLevel parameter when calling Printer,
+allowing implementations to customize each printed message depending on the method from which
+it originated. However, it is common practice to group together certain methods and treat their
+output similarly, in four broad categories. This table summarizes these common groupings:
+
+<table>
+  <thead>
+    <tr>
+      <th>Grouping</th>
+      <th>{{console}} method</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td>log</td>
+    <td>
+      <code>
+        {{console/log()}}, {{console/trace()}}, {{console/dir()}}, {{console/dirxml()}},
+        {{console/group()}}, {{console/groupCollapsed()}}, {{console/debug()}}
+      </code>
+    </td>
+    <td>
+      A generic log
+    </td>
+  </tr>
+  <tr>
+    <td>info</td>
+    <td><code>{{console/count()}}, {{console/info()}}, {{console/timeEnd()}}</code></td>
+    <td>
+      An informative log
+    </td>
+  </tr>
+  <tr>
+    <td>warn</td>
+    <td><code>{{console/warn()}}</code></td>
+    <td>
+      A log warning the user of something indicated by the message
+    </td>
+  </tr>
+  <tr>
+    <td>error</td>
+    <td><code>{{console/error()}}, {{console/assert()}}</code></td>
+    <td>
+      A log indicating an error to the user
+    </td>
+  </tr>
+</table>
+
+These groupings are meant to document common practices, and do not constrain implementations from
+providing special behavior for each method, as in the following examples:
+
+<div class="example" id="timeEnd-formatting">
+  Formatting applied to output produced by calls to {{console/timeEnd()}} may be unique to this
+  method.
+
+  <img alt="A demonstration of timeEnd and info formatting differences" src="images/timeEnd-formatting.png">
+</div>
+
+<div class="example" id="count-output">
+  Calls to {{console/count()}} may not always print new output.
+  <p class="XXX">TODO: A picture of this in the Edge console would be nice.</p>
+</div>
+
+<h4 id="printer-ux-innovation">Printer User Experience innovation</h4>
+
+Since Printer is implementation-defined, it is common to see UX innovations in its implementations.
+The following is a non-exhaustive list of UX enhancements that may appear in existing
+implementations:
```

Comment:
Ok I think that makes sense. With that said, I revised the `count` para to say:

> Calls to {{console/count()}} **might** not always print new output, but instead could update previously-output counts.

(not actually bold in spec). I think that *might* be a little better too, since that word isn't mentioned in https://infra.spec.whatwg.org/#conformance